### DISCUSSION TITLE
Is github issues using swr for reactions & comments live update

### DISCUSSION THREAD

**Discussion by liho00** [vercel/swr#1545] [Q&A]
How github issues reactions live update the likes count & comments in the webpage on any devices opened the page, may i know what stacks or technology being used in the github issues frontend (izit using swr?) & backend? 

Its like magic to me, that github able to push the update of content to the issue page without websocket & sse. How does it identify the client & update the client’s webpage?

my question is does the github using swr to achieve live comments update?

**Comment by shuding** [comment]
I can't answer it obviously 😄, but since github.com is not based on React it can't be using the SWR library. My guess is it uses some long polling mechanism.

**Reply by liho00** [reply]
how did u know github.com is not using React? Besides, i did inspect the network requestion from what i mentioned. Its not using long polling mechanism, since there is no long polling request shown in the network request.
